.github/workflows/docker_image.yml

```diff
@@ -0,0 +1,63 @@
+# This workflow should be triggered when a release is done (workflow_call)
+# And also need to be able to run manually (workflow_dispatch). It uses
+# the current version from poetry as a tag for the docker container and
+# pushes the container to the GitHub Container Registry
+# (ghcr.io/deltares/d-ecoimpact:latest).
+
+name: "Publish to GHCR"
+
+on:
+  workflow_call:
+    inputs:
+      project_version:
+        required: true
+        default: false
+        type: string
+  workflow_dispatch:
```

Comment:
The package version should be retrieved from the pyproject.toml (from poetry). In order to always keep these versions synched. So preferably we don't want an input in theis file.. but because this workflow is also triggered from another workflow (release). Here the version is upgraded. But in this folow up workflow the previous version is used. I think it might be a timing problem, so to be sure I passed the version through from the release workflow to this workflow. So No I don't want an input :)